src/main/java/org/isf/menu/gui/Login.java

```diff
@@ -58,6 +55,14 @@
 import org.isf.utils.time.TimeTools;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.util.FileCopyUtils;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
static imports are placed first in the list of imports please

Reply:
fixed!!